.github/workflows/build.yml

```diff
@@ -26,3 +26,12 @@ jobs:
         with:
           name: omero-ms-image-region ${{ matrix.java }}
           path: build/distributions/*
+      - name: Set ARTIFACTORY_URL
+        run: |
+          echo "ARTIFACTORY_URL=https://artifacts.openmicroscopy.org/artifactory/ome.releases" >> $GITHUB_ENV
+      - name: Publish
+        if: startsWith(github.ref, 'refs/tags') && github.repository_owner == 'ome' && matrix.java == 11
+        run: ./gradlew publish
```

Comment:
Is the goal to publish the JAR or the micro-service bundle? Should the groupId coordinates be updated to reflect the ownership of this fork?

Reply:
I started to work on https://github.com/ome/ansible-role-omero-ms-image-region/pull/4 afterwards, the zip is required so I will look into the option of attaching the zip to the tag instead